.github/workflows/ci.yml

```
@@ -91,6 +91,23 @@ jobs:
       - name: Run tests
         run: cargo test --release -p cast
 
+  scarbfmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: software-mansion/setup-scarb@v1.3.2
+      - name: Check cairo files format
+        run: |
+          output=$(find . -type f -name "Scarb.toml" -execdir sh -c '
+              echo "Running \"scarb fmt\" in directory: $PWD"
+              scarb fmt --check
+          ' \;)
+          echo "$output"
+          if grep -iq "Diff" <<< "$output"; then
+              exit 1
+          fi
+              exit 0
```

```suggestion
          exit 0
```
nitpick, but let's keep it properly formatted :+1: